composer/trainer/trainer.py

```diff
@@ -3031,11 +3031,12 @@ def _eval_loop(
                                     outputs = self.state.outputs
 
                                 for metric in metrics.values():
-                                    self._original_model.update_metric(
+                                    metric_outputs = self._original_model.update_metric(
```

Comment:
Model doesn't require returning anything per type hints?

Reply:
currently its only in hf model: https://github.com/mosaicml/composer/pull/2977/files/25f1872fc109d00138b8a46dfed2c06aa89160a9#diff-e61f9af34de66238d99995ebdc5d2645cea3c7ca7c303ac006239a720c5b620cR533

afaik we only do evals w/ huggingface models? Given the implementations here: https://github.com/mosaicml/composer/blob/25f1872fc109d00138b8a46dfed2c06aa89160a9/composer/models/huggingface.py#L488

Reply:
Do we need to return this? Can we just pull the metric from state?

Reply:
We are returning this in order to set the value in state.